vulcan/data/_messagebox.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+from typing import AsyncIterator, List, Union
+
+from related import IntegerField, StringField, immutable
+
+from .._endpoints import DATA_MESSAGEBOX
+from ..model import Serializable
+
+
+@immutable
+class Messagebox(Serializable):
```

Comment:
Wouldn't `MessageBox` look nicer? Those are actually two words, so following the PascalCase convention, both should start with an uppercase letter.